src/config/settings.py

```diff
@@ -67,10 +69,19 @@
     "django.middleware.common.CommonMiddleware",
     "django.middleware.csrf.CsrfViewMiddleware",
     "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django_otp.middleware.OTPMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
 
+DJANGO_OTP_ADMIN = strtobool(os.getenv("DJANGO_OTP_ADMIN", "true"))
+if DJANGO_OTP_ADMIN:
+    # Use OTP admin
+    INSTALLED_APPS.append("config.admin.OTPAdminConfig")
+else:
+    # Use Default admin
+    INSTALLED_APPS.append("django.contrib.admin")
```

Comment:
@Uxio0 Wondering what is the best way to test this. Overriding the settings in the tests won't work since settings are loaded when the Django initialises.

It looks like we need a test that loads Django after setting the environment variable `DJANGO_OTP_ADMIN`. And then we can check the installed applications.

Reply:
You can enable it always for tests, does it breaks your admin tests?

Reply:
The tests don't break but the coverage decreased because we are not testing this `if/else` branch 😅 

I think for this it would require a test where we set an environment variable and programatically launch Django. This can be probably be tackled later if required.

Reply:
If the only problem is coverage, you can use: https://coverage.readthedocs.io/en/coverage-4.3.3/excluding.html#excluding-code-from-coverage-py I think it's for cases like this one. I think trying to test this can give us more trouble than the problem we are trying to fix